radixdlt/babylon-wallet-android
# Add affected entities info to bdfs error

## Description
Adds affected account/personas info to error dialog


## How to test

1. restore wallet from profile file containing bdfs error - please reach to me if you need one.
2. Run the wallet and verify that error is shown and entities are listed

## Screenshot

<img src="https://github.com/radixdlt/babylon-wallet-android/assets/118203440/6e9c7db7-6976-4834-a80d-08141a1adc6b" width=30%>

## PR submission checklist
- [ ] I have tested bdfs error
